src/products/event/template.tsx

```diff
@@ -330,17 +331,17 @@ export const Event = ({
         <h2 className="vads-u-font-size--h3 vads-u-margin-bottom--0">
           Other VA events
         </h2>
-        {administration?.id != 7 &&
+        {administrationId != 7 &&
           listingOffice != 'Outreach and events' &&
-          administration?.name && (
+          administrationName && (
             <p>
               <va-link
                 href={listing}
                 onClick={() =>
                   recordEvent({ event: 'nav-secondary-button-click' })
                 }
                 id="see-more-events"
-                text={`Browse the ${administration.name} events calendar`}
+                text={`Browse the ${administrationName} events calendar`}
               ></va-link>
```

Comment:
The solution for simplifying was based on my earlier evaluation that only the id was ever used in the site, but I was wrong. All the changes were based on that disproven assumption. But now the work day is over, so I'm going to leave this here.

Reply:
Because of this, I created [an alternate path](https://github.com/department-of-veterans-affairs/next-build/pull/1048), which is what @cvalarida had suggested in the first place.